src/features/cookie.js

```diff
@@ -176,16 +211,24 @@ export default class CookieFeature extends ContentFeature {
 
     init (args) {
         if (args.cookie) {
+            const trackerLookup = cookiePolicy.trackerLookup
             cookiePolicy = args.cookie
+            cookiePolicy.trackerLookup = trackerLookup
```

Comment:
Can we not assign `trackerLookup` to a different global, rather than doing this copy?